### DISCUSSION TITLE
`createDocument` `correlationId` nékül

### DISCUSSION THREAD

**Discussion by rrd108** [nav-gov-hu/M2M#49] [Kérdések] [answered]
Ha jól látom a `createDocument` endpoint 400-as hibára fut `correlationId` nélkül.

A leírás szerint ez egy opcionális adat, de nekem nem sikerült enélkül kommunikálni az API-val ezen a végponton.

**Comment by TamasPill** [comment, marked as answer]
Ez valóban egy probléma, javítani fogjuk. 
Addig a correlationId töltésével gond nélkül használható.

**Reply by rrd108** [reply]
köszi a választ
